### PR TITLE
use column table_schema instead of schema_name

### PR DESCRIPTION
**Pull request by jodok**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ Changes for crash
 Unreleased
 ==========
 
+ - Use column ``table_schema`` instead of ``schema_name`` in order to be
```
Comment (chaudum) [inline CHANGES.txt]: not a user facing change. changelog entry can be omitted.

File: src/crate/crash/command.py [hunk 2]
```diff
@@ -235,9 +235,9 @@ def exit(self):
     @noargs_command
     def _show_tables(self, *args):
         """ print the existing tables within the 'doc' schema """
-        self._exec("""select format('%s.%s', schema_name, table_name) as name
+        self._exec("""select format('%s.%s', table_schema, table_name) as name
```
Comment (chaudum) [inline src/crate/crash/command.py]: this command should check for the server version and then execute the correct statement accordingly

File: src/crate/crash/sysinfo.py [hunk 3]
```diff
@@ -39,15 +39,15 @@ class SysInfoCommand(object):
...
+                         from sys.shards """,
 
                      """ select count(1) as number_of_nodes
                            from sys.nodes """,
 
                      """ select count(distinct(table_name))
                                  as number_of_tables
                          from information_schema.tables
-                         where schema_name
+                         where table_schema
```
Comment (chaudum) [inline src/crate/crash/sysinfo.py]: same here

File: src/crate/crash/command.py (hunk 2 above)
Comment (jodok) [inline src/crate/crash/command.py]: updated the check. however i had to define the TABLE_SCHEMA_MIN_VERSION twice, and also have the feeling that i needed to duplicate too much SQL code :)
let me know if you have suggestions on how to better structure it

File: CHANGES.txt [hunk 4]
```diff
@@ -5,6 +5,7 @@ Changes for crash
 Unreleased
 ==========
 
+
```
Comment (bogensberger) [inline CHANGES.txt]: blank line

File: CHANGES.txt [hunk 5]
```diff
@@ -32,7 +33,6 @@ Unreleased
 
  - Require Pygments version greater or equal 2.0
 
-
```
Comment (bogensberger) [inline CHANGES.txt]: blank line

File: CHANGES.txt [hunk 6]
```diff
@@ -5,6 +5,7 @@ Changes for crash
 Unreleased
 ==========
```
Comment (bogensberger) [inline CHANGES.txt]: Add changes entry

File: src/crate/crash/command.py [hunk 7]
```diff
@@ -238,9 +239,15 @@ def exit(self):
...
-                      from information_schema.tables
-                      where schema_name not in ('sys','information_schema', 'pg_catalog')""")
+        if self.connection.lowest_server_version >= TABLE_SCHEMA_MIN_VERSION:
+            self._exec("""select format('%s.%s', table_schema, table_name) as name
+                          from information_schema.tables
+                          where table_schema not in ('sys','information_schema', 'pg_catalog')""")
+        else:
+            self._exec("""select format('%s.%s', schema_name, table_name) as name
+                          from information_schema.tables
+                          where schema_name not in ('sys','information_schema', 'pg_catalog')""")
```
Comment (bogensberger) [inline src/crate/crash/command.py]: ```"""select format('%s.%s', {schema}, table_name) as name ...""".format(schema="table_schema" if self.connection.lowerst_server_version >=  TABLE_SCHEMA_MIN_VERSION else "schema_name")```
should do the job without repeating the query

File: src/crate/crash/command.py [hunk 8]
```diff
@@ -238,9 +239,15 @@ def exit(self):
...
+
```
Comment (bogensberger) [inline src/crate/crash/command.py]: blank line

File: src/crate/crash/sysinfo.py [hunk 9]
```diff
@@ -31,15 +31,16 @@
 from .printer import ColorPrinter
 
 
+
```
Comment (bogensberger) [inline src/crate/crash/sysinfo.py]: blank line

File: src/crate/crash/sysinfo.py [hunk 10]
```diff
@@ -101,7 +115,11 @@ def _sys_info(self):
     def _cluster_info(self, result):
         rows = []
         cols = []
-        for query in SysInfoCommand.CLUSTER_INFO:
+        cluster_info_queries = SysInfoCommand.CLUSTER_INFO
+        if self.cmd.connection.lowest_server_version >= TABLE_SCHEMA_MIN_VERSION:
+          cluster_info_queries = SysInfoCommand.CLUSTER_INFO_TABLE_SCHEMA
```
Comment (bogensberger) [inline src/crate/crash/sysinfo.py]: wrong indent? (only two spaces instead of four)

File: src/crate/crash/sysinfo.py [hunk 11]
```diff
@@ -50,6 +50,19 @@ class SysInfoCommand(object):
                          where schema_name
                          not in ('information_schema', 'sys') """ ]
 
+    CLUSTER_INFO_TABLE_SCHEMA = [ """ select count(distinct(id)) as number_of_shards,
```
Comment (kovrus) [inline src/crate/crash/sysinfo.py]: probably keeping the same of queries a little bit too much, we can just change the last item in the list(modify, append, etc)

**Comment by m-kharbat** [comment]
for the change log : https://github.com/crate/crash/pull/152#discussion_r82549898

File: src/crate/crash/command.py [hunk 12]
```diff
@@ -238,9 +242,10 @@ def exit(self):
     @noargs_command
     def _show_tables(self, *args):
         """ print the existing tables within the 'doc' schema """
-        self._exec("""select format('%s.%s', schema_name, table_name) as name
+        self._exec("""select format('%s.%s', {schema}, table_name) as name
                       from information_schema.tables
-                      where schema_name not in ('sys','information_schema', 'pg_catalog')""")
+                      where table_schema not in ('sys','information_schema', 'pg_catalog')"""
+                      .format(schema="table_schema" if self.connection.lowerst_server_version >= TABLE_SCHEMA_MIN_VERSION else "schema_name"))
```
Comment (kovrus) [inline src/crate/crash/command.py]: pep8 line length

File: src/crate/crash/command.py [hunk 13]
```diff
@@ -238,9 +242,11 @@ def exit(self):
     @noargs_command
     def _show_tables(self, *args):
         """ print the existing tables within the 'doc' schema """
-        self._exec("""select format('%s.%s', schema_name, table_name) as name
+        self._exec("""select format('%s.%s', {schema}, table_name) as name
                       from information_schema.tables
-                      where schema_name not in ('sys','information_schema', 'pg_catalog')""")
+                      where table_schema not in ('sys','information_schema', 'pg_catalog')"""
+                      .format(schema="table_schema" if self.connection.lowerst_server_version\
```
Comment (kovrus) [inline src/crate/crash/command.py]: can you extract it from format

**Pull request merged**